src/Symfony/Bundle/FrameworkBundle/Resources/config/annotations.xml

```diff
@@ -19,7 +19,12 @@
             <argument /><!-- Debug-Flag -->
         </service>
 
+        <service id="annotations.php_file_cache" class="Doctrine\Common\Cache\PhpFileCache" public="false">
+            <argument /><!-- Cache-Directory -->
+        </service>
+
         <service id="annotations.file_cache_reader" class="%annotations.file_cache_reader.class%" public="false">
+            <deprecated>The "%service_id% service is deprecated since 2.8 and will be removed in 3.0."</deprecated>
```

Comment:
a private service can still be used by any other service. It is not about restricting access to it, but about giving a hint to the optimizer that this service is private to the container (i.e. the container can give it to other services, but not to the outside through ``get``, and so it can inline it)